omod/src/main/webapp/pages/results.gsp

```diff
@@ -156,7 +217,9 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
         patients.push("${prettify(ui.encodeHtmlContent(it.name))}");
     </script>
     <tr>
-        <td style="display: none">rule_${it.rule.ruleId}</td>
+        <td class="hidden">rule_${it.rule.ruleId}</td>
+        <td class="hidden">${it.patient.patientIdentifier}</td>
```

Comment:
This should be a check if you have a patient or a program - not two separate columns, later we may add a third type of validation class so this has to be extensible